tests/functional_tests/tools/virtualenv/pkam_virtualenv.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+FULL_PATH_TO_SCRIPT="$(realpath "${BASH_SOURCE[0]}")"
+SCRIPT_DIRECTORY="$(dirname "$FULL_PATH_TO_SCRIPT")"
+cd "$SCRIPT_DIRECTORY"
+
+CONTAINER_NAME="virtualenv"
+
+sudo docker exec "$CONTAINER_NAME" supervisorctl start pkamLoad
```

Comment:
don't put sudo in the script. Most systems should have configured docker so that it doesn't need super user permissions.

Reply:
Done in https://github.com/atsign-foundation/at_c/pull/476/commits/fb2e8a34edb142da49f59e983b306a3641f6e57a